common_sleep.Rmd

```diff
@@ -173,16 +176,43 @@ ggplot(melt(cors), aes(Var1,Var2, fill = value)) +
        y = "reaction on day #")
 ```
 
-We used the Shapiro-Wilk test to check for normality of the reaction times per day.  
+We used the Shapiro - Wilk test to check for the normality of the reaction times per day. The test revealed a non normal distribution of day 9. Thus, we performed the spearman correlation method instead of pearson to check for a correlation of the reaction times between days. 
 
-The test revealed a non normal distribution of day 9. Thus, we performed the spearman correlation method instead of pearson to check for a correlation of the reaction times between days. 
```

Comment:
We don't have a correlation matrix anymore. Just a heatmap. we should describe it in more general terms.